cmake/Toolchain-Autotalks.cmake

```diff
@@ -0,0 +1,15 @@
+set(CMAKE_SYSTEM_NAME "Linux")
+set(CMAKE_SYSTEM_PROCESSOR "arm")
+set(CMAKE_C_COMPILER "/opt/poky-craton2/2.6.2/sysroots/x86_64-pokysdk-linux/usr/bin/arm-poky-linux-gnueabi/arm-poky-linux-gnueabi-gcc")
```

Comment:
Is */opt/poky-craton2* a path you have chosen or is it common for all Autotalks SDK installations?
Anyhow, I suggest adding a POKY_CRATON2 CMake variable with "/opt/poky-craton2/2.6.2" so one can easily switch to another toolchain (version).

Reply:
This is actually a path that is created when installing the toolchain (Autotalks have they own install script), so it should be common for all installations. I have added the POKY_CRATON2 variable nonetheless.